examples/custom/01-parameter-passing/cf_demo1.c

```diff
@@ -0,0 +1,110 @@
+#include "pf_all.h"      /* lots of stuff */
+#include "cf_helpers.h"  /* to_C_string */
+#include<errno.h>        /* errno         */
+#include<limits.h>       /* PATH_MAX */
+#include<stdlib.h>       /* malloc */
+#include<stdio.h>        /* asprintf, sprintf */
+#include<string.h>       /* strerror */
+#include<sys/stat.h>     /* struct stat, stat */
+
+/* 
+ * put forward declarations here if necessary
+*/
+
+
+/****************************************************************
+** Step 1: Put your own special glue routines here
+**     or link them in from another file or library.
+****************************************************************/
+
+/* exported functions */
+
+static cell_t f4711( cell_t Val )
+{/* a quick way to check that custom words are available
+ */
+    return 11 + 47*Val;
+}
+
+static cell_t FileInfo( cell_t path_caddr, cell_t path_len ) {
+	char* path = to_C_string( path_caddr, path_len );
```

Comment:
This function uses tabs for indentation.
Please use 4 spaces. See:
https://github.com/philburk/pforth/wiki/Style-Guide